biaspaltsau_a/14.Ansible.Workshop/roles/db/templates/my.cnf.j2

```diff
@@ -0,0 +1,5 @@
+[client]
```

Comment:
I do not remember this in our workshop, please remove

Reply:
corrected